### PR TITLE
Some fixed and added documentation about async jobs, projections, bus…

### LINKED ISSUE #57: Documentation
Coming from #47 

Some people has requested documentation or faq about design decisions and also about how to continue adding things to the project and why that way. 

So, to document:

#### Command, Query and Event bus

- [x] Why Tactician for Command/Query Bus? (A: because of the return at resolver level)
- [x] How to add a new use case (Command or Query)  

#### Event Store
- [ ] How to replay events?

#### Read Model
- [x] How to build a projection?

#### Async Event subscribers
- [x] How do Symfony/RabbitMQ/ElasticSearch work together?

### PR DESCRIPTION
**Pull request by jorge07**
… and use case

Fixes #57

### CODE REVIEW

**Comment by Lutacon** [comment]
Awesome PR!!

Could you remove a typo in makefile? https://github.com/jorge07/symfony-4-es-cqrs-boilerplate/blob/master/makefile#L37
It says `.PHONY: layers` it should be `.PHONY: layer`

File: doc/GetStarted/UseCases.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+`docker-compose exec php sh -l`
+
+Execute:
+
+`./bin/console app:echo`
+
+And you should see: `LOOL`
+
+And that's all with 0 config thanks to Symofny 4!
```
Comment (Lutacon) [inline doc/GetStarted/UseCases.md]: Typo here in "Symofny 4"

**Comment by jorge07** [comment]
@Lutacon @pascal08 @danistc I'll wait for you review to merge this ;)

**Comment by pascal08** [comment]
Looks very good! I will read this thoroughly tonight!

**Comment by pascal08** [comment]
Read it all the way through. Very nice summary!

**Pull request merged**